darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -63,19 +71,29 @@ def __init__(
                 `regression_train_num_samples will be passed only to the probabilistic ones.
             ..
         regression_train_samples_reduction
-            If `forecasting models` are probabilistic and `regression_train_num_samples` > 1, method used to
+            If `forecasting_models` are probabilistic and `regression_train_num_samples` > 1, method used to
             reduce the samples before passing them to the regression model. Possible values: "mean", "median"
             or float value corresponding to the desired quantile. Default: "median"
+        train_forecasting_models
+            If set to `False`, the `forecasting_models` are not retrained when calling `fit()` (only supported
+            if all the `forecasting_models` are pretrained `GlobalForecastingModels`). Default: ``True``.
+        train_using_historical_forecasts
+            If set to `True`, use `historical_forecasts()` to generate the forecasting models' predictions used to
+            train the regression model in `fit()`. Available when `forecasting_models` contains only
+            `GlobalForecastingModels` with `output_chunk_length` multiple of `regression_train_n_points` and
+            recommended when `regression_train_n_points` is considerably greater than `output_chunk_length`.
```

Comment:
I believe that ocl doesn't have to be multiple of train_n_points anymore, right?
```suggestion
            train the regression model in `fit()`. Available when `forecasting_models` contains only
            `GlobalForecastingModels`. Recommended when `regression_train_n_points` is greater than 
            `output_chunk_length` of the underlying `forecasting_models`.
```


```suggestion
            `GlobalForecastingModels`. Recommended when `regression_train_n_points` is greater than `output_chunk_length` of the underlying `forecasting_models`.
```